### PR TITLE
Appx.C Fuzz Vectors - v4 Migration Cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR covers issue #92 

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix links 
- Fix linting issues

### CODE REVIEW

**Comment by kingthorin** [comment]
I think the nowiki tags on this content can also be removed

**Comment by rejahrehim** [comment]
> I think the nowiki tags on this content can also be removed

Done

**Comment by kingthorin** [comment]
@rejahrehim please open a PR adding yourself to the credits:
https://github.com/OWASP/OWASP-Testing-Guide-v5/blob/master/document/1_Frontispiece/1_Frontispiece.md#v5-reviewerseditors

**Comment by rejahrehim** [comment]
> @rejahrehim please open a PR adding yourself to the credits:
> https://github.com/OWASP/OWASP-Testing-Guide-v5/blob/master/document/1_Frontispiece/1_Frontispiece.md#v5-reviewerseditors

Where should I add?
Authors or reviewers?

**Comment by kingthorin** [comment]
Reviewers/Editors, makes sense at this point. If you actually author new content later then move or duplicate it (I guess)  :man_shrugging:  :grin:

File: document/Appx.C_Fuzz_Vectors/Appx.C_Fuzz_Vectors.md [hunk 1]
```diff
@@ -1,91 +1,93 @@
...
 
-Selecting “8302fa3b” as a part of the request to be fuzzed against the set hexadecimal alphabet (i.e. {0,1,2,3,4,5,6,7,8,9,a,b,c,d,e,f}) falls under the category of recursive fuzzing. This would generate a total of 16\^8 requests of the form:
+Selecting “8302fa3b” as a part of the request to be fuzzed against the set hexadecimal alphabet (i.e. {0,1,2,3,4,5,6,7,8,9,a,b,c,d,e,f}) falls under the category of recursive fuzzing. This would generate a total of 16^8 requests of the form:
 
-    <nowiki>http://www.example.com/00000000</nowiki>
-    <nowiki>...</nowiki>
-    <nowiki>http://www.example.com/11000fff</nowiki>
-    <nowiki>...</nowiki>
-    <nowiki>http://www.example.com/ffffffff</nowiki>
+`http://www.example.com/00000000`
```
Comment (ThunderSon) [inline document/Appx.C_Fuzz_Vectors/Appx.C_Fuzz_Vectors.md]: Note that your formatting broke the new lines. You can put them in triple backticks, or put a new line between them. Same goes for the rest of the document.

Comment (rejahrehim) [reply document/Appx.C_Fuzz_Vectors/Appx.C_Fuzz_Vectors.md]: Noted.

Comment (rejahrehim) [reply document/Appx.C_Fuzz_Vectors/Appx.C_Fuzz_Vectors.md]: Added newline in between all code tags.

**Pull request merged**